solutions/CCC 20-escape.mdx

```diff
@@ -14,232 +14,40 @@ So we essentially have a directed graph with vertices $1\ldots 10^6$ where there
 Thus, we can simply start DFSing from vertex $1$ and mark all the vertices that we visit in the boolean array `done`. If we set `done[N*M]=true`, then a path exists, and we can terminate after printing "yes." Note that in the code below, `todo[x]` denotes the adjacency list of `x`.
 
 ```cpp
-#include <bits/stdc++.h>
```

Comment:
stdc++ is fine.

Reply:
I mean, it only adds one line to the code.

Reply:
use the template agreed upon in #803